src/components/screens/timebank-screen.tsx

```diff
@@ -1,159 +1,38 @@
       setSelectedEmployeeId(loggedInPerson.id);
     }
   }, [loggedInPerson]);
 
-  useEffect(() => {
-    getPersonDailyEntries();
-  }, [persons]);
-
-  useEffect(() => {
-    if (selectedPerson) {
-      getPersonDailyEntriesForPieChart(selectedPerson);
-    }
-  }, [persons, selectedPerson]);
-
-  /**
-   * Gets  selectedPerson's total time data.
-   *
-   * @param selectedPersonId selected person id
-   */
-
-  /**
-   * Gets person's total time data.
-   */
-  const getPersonTotalTime = async (selectedPersonId: number) => {
-    setLoading(true);
-    if (selectedPersonId) {
-      try {
-        const fetchedPersonTotalTime = await personsApi.listPersonTotalTime({
-          personId: selectedPersonId,
-          timespan: timespan || Timespan.ALL_TIME,
-          before: DateTime.now().minus({ days: 1 }).toJSDate()
-        });
-        setPersonTotalTime(fetchedPersonTotalTime[0]);
-      } catch (error) {
-        setError(`${strings.error.totalTimeFetch}, ${error}`);
-      }
-    }
-    setLoading(false);
-  };
-
-  /**
-   * Gets the logged in person's daily entries.
-   */
-  const getPersonDailyEntries = async () => {
-    if (loggedInPerson) {
-      try {
-        const fetchedDailyEntries = await dailyEntriesApi.listDailyEntries({
-          personId: loggedInPerson.id
-        });
-        setDailyEntries(fetchedDailyEntries);
-        setPersonDailyEntry(
-          fetchedDailyEntries.find((item) => item.date <= new Date() && item.logged)
-        );
-      } catch (error) {
-        setError(`${strings.error.dailyEntriesFetch}, ${error}`);
-      }
-    }
-  };
-
-  /**
-   * Gets daily entries for the selected employee for the pie chart.
-   *
-   * @param selectedPerson daily entries
-   */
-  const getPersonDailyEntriesForPieChart = async (selectedPerson: Person) => {
-    if (selectedPerson) {
-      try {
-        const fetchedDailyEntries = await dailyEntriesApi.listDailyEntries({
-          personId: selectedPerson.id
-        });
-        setDailyEntries(fetchedDailyEntries);
-        setPersonDailyEntry(
-          fetchedDailyEntries.find((item) => item.date <= new Date() && item.logged)
-        );
-      } catch (error) {
-        setError(`${strings.error.dailyEntriesFetch}, ${error}`);
-      }
-    }
-  };
-
-  /**
-   * Changes the displayed daily entry in the pie chart via the Date Picker.
-   *
-   * @param selectedDate selected date from DatePicker
-   */
-  const handleDailyEntryChange = async (selectedDate: DateTime) => {
-    try {
-      const fetchedDailyEntries = await dailyEntriesApi.listDailyEntries({
-        personId: selectedEmployeeId
-      });
-      setDailyEntries(fetchedDailyEntries);
-      setPersonDailyEntry(
-        fetchedDailyEntries.find((item) => item.person === selectedEmployeeId && DateTime.fromJSDate(item.date).toISODate() === selectedDate.toISODate())
-      );
-    } catch (error) {
-      setError(`${strings.error.dailyEntriesFetch}, ${error}`);
-    }
-  };
-
```

Comment:
Is this change related to some re-rendering issues you fixed for another feature?

Reply:
Yes